pkg/sql/schemachanger/sctest/framework.go

```diff
@@ -453,6 +454,8 @@ func (e *stageExecStmt) Exec(
 				return strconv.Itoa(stageVariables.successfulStageCount)
 			case "stageKey":
 				return strconv.Itoa(stageVariables.stage.AsInt() * 1000)
+			case "phase":
+				return stageVariables.stage.phase.String()
```

Comment:
what is this for? Is it even ever used?

Reply:
Ah that was part of another PR that I forgot to cleanup. Removed.